sycl/plugins/level_zero/usm_allocator.cpp

```diff
@@ -475,17 +489,17 @@ class Bucket {
   void updateStats(int InUse, int InPool);
 
   // Print bucket statistics
-  void printStats();
+  void printStats(bool &TitlePrinted, SystemMemory::MemType MT);
 
 private:
   void onFreeChunk(Slab &, bool &ToPool);
 
+  void decrementPool(bool &FromPool);
```

Comment:
Please document all methods and their arguments

Reply:
OK, will do for methods added in this change.